server/matches.py

```diff
@@ -81,4 +81,33 @@ def getConversationIds(currentUser, friendUser):
      except mysql.connector.Error as err:
         return {"response": err.msg }
 
+def unmatch(currentUser, friendUser):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            change_potentialMatch = f"UPDATE PotentialMatch SET matchDecision = 0 WHERE currentUser = %s AND shownUser = %s"
```

Comment:
the potential match table will have 2 entries for a match, one where the currentUser and friendUser are swapped, do both rows in the table get updated?